Functions/QuickScript/SteamScript/script.js

```diff
@@ -20,10 +21,59 @@ SteamScript.prototype.constructor = SteamScript;
 SteamScript.prototype.appId = function(appId) {
     this._appId = appId;
     this._applicationHomepage = "http://store.steampowered.com/app/" + appId;
-    this._executableArgs = "steam://rungameid/" + this._appId;
+    this._executableArgs = ["-silent", "steam://rungameid/" + this._appId];
     return this;
 };
 
+SteamScript.prototype.getBytesToDownload = function(wine) {
+    // wait until download started
+    while (!fileExists(wine.prefixDirectory + "/drive_c/" + wine.getProgramFiles() + "/Steam/steamapps/appmanifest_" + this._appId + ".acf"))
+    {
```

Comment:
You can also create a JavaScript Function Script (Threading) and define a sleep function inside